scripts/Muon/GUI/ElementalAnalysis2/grouping_widget/ea_grouping_table_widget_presenter.py

```diff
@@ -133,11 +133,6 @@ def handle_update(self, update_model):
             self.notify_data_changed()
             return
 
-        try:
-            self.update_model_from_view()
-        except ValueError as error:
-            self._view.warning_popup(error)
-
```

Comment:
Why has this try and exception been removed?

Reply:
information on rebinned workspaces and tables produced from groups are erased when self.update_model_from_view method is called